ansible/www-standalone/tasks/site-setup.yaml

```diff
@@ -110,11 +110,9 @@
     dest: /etc/crontab
     line: "{{ item }}"
   with_items:
-    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh nodejs'
-    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh iojs'
-    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
-    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh iojs'
-    - '*  *    * * *   root    /home/nodejs/cdn-purge.sh'
+    - '*/30 *    * * *   dist    /home/staging/tools/promote/promote_nightly.sh nodejs'
```

Comment:
This checks for auto-promotable assets and moves them from staging to dist. This is everything but releases that get signed, so nightlies, v8-canary, RC, test, and others. Release machines upload resources to staging and every 5 minutes this script will look for which ones can be promoted and will do so. If at least one thing was promoted, it will queue a request for a cache purge (it doesn't perform the purge itself). So it's theoretically possible to purge CDN cache every 5 minutes if we have a new resource to promote every 5 minutes.

Slowing to every 30 minutes. Which means we're more likely to promote most of any of these drops in one or two goes. But it slows down the time between the drops and when they might show up. This might get frustrating for someone who's doing an RC, for example, watching their builds complete but not show up on nodejs.org so they can announce them. So there's a balance here.